packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html

```diff
@@ -0,0 +1,23 @@
+<oui-tile
+    data-heading="{{ 'telephony_line_softphone_download_links_title' | translate }}"
+>
+    <oui-tile-definition
+        data-term="{{ 'telephony_line_softphone_download_links_description_label' | translate }}"
+    >
+    </oui-tile-definition>
+</oui-tile>
```

Comment:
For the download part it ask an oui-collapsible not a tile.

Reply:
Done, use oui collapsible everywhere with custom one (not clickable)